File: W1/abcz2.c
#include <stdio.h>

int main() {
    int x;
    int a = 1;
    
    scanf("%d", &x);
    
    if ( x % 2 == 0 ) {
        printf("alpha\n");
        a = 0;
    }
    if ( x % 3 == 0 ) {
        printf("bravo\n");
        a = 0;
    }
    if ( x % 5 == 0 ) {
        printf("charlie\n");
        a = 0;
    }
    if ( a ) {
        printf("zulu\n");
    }
    // if ( a == 1 ) {
    //     printf("zulu\n");
    // }
    
    return 0;
}

File: PHP/abcz2.php
<?php
    $val = fgets(STDIN);
    $isZulu = true;
    
    if ( $val % 2 == 0 ) {
        echo "alpha" . PHP_EOL;
        $isZulu = false;
    }
    if ( $val % 3 == 0 ) {
        echo "bravo" . PHP_EOL;
        $isZulu = false;
    }
    if ( $val % 5 == 0 ) {
        echo "charlie" . PHP_EOL;
        $isZulu = false;
    }
    if ( $isZulu ) {
        echo "zulu" . PHP_EOL;
    }
?>
